app/views/idv/shared/_mini_logo.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= image_tag(
+      asset_url('logo.svg'),
+      alt: APP_NAME,
+      class: 'display-block margin-left-auto margin-right-auto margin-top-1 margin-bottom-1',
+      height: 15,
+      width: 110,
+      skip_pipeline: true,
```

Comment:
Generally speaking, mostly you should never skip pipeline, since it serves the files out of the Rails server directly, bypassing our CDN.

I'd guess this was inherited from how it's done in BarcodeComponent, which renders an image as a `data:` URL in the application, so the asset pipeline would not be relevant to serving that image.